src/usermodel.cpp

```diff
@@ -367,10 +394,15 @@ void UserModel::removeGroups(int row, const QStringList &groups)
         return;
 
     createInterface();
-    auto call = m_dBusInterface->asyncCall(QStringLiteral("removeFromGroups"), (uint)user.uid(), groups);
-    auto *watcher = new QDBusPendingCallWatcher(call, this);
-    connect(watcher, &QDBusPendingCallWatcher::finished,
-            this, std::bind(&UserModel::removeFromGroupsFinished, this, std::placeholders::_1, user.uid()));
+    const int uid = user.uid();
+    auto response = m_dBusInterface->call(QStringLiteral("removeFromGroups"), (uint)uid, groups);
+    response->onError([this, uid](const QDBusError &error) {
+        emit removeGroupsFailed(m_uidsToRows.value(uid), getErrorType(error));
+        qCWarning(lcUsersLog) << "Adding user to groups failed:" << error;
```

Comment:
Would you mind fixing this old mistake? It says "Adding" although this is clearly about removing.

Reply:
Done

Reply:
Thanks!